### PR TITLE
fix(VProgressCircular): dark theme background color clash

### LINKED ISSUE #12149: [Bug Report] VProgressCircular doesn't show unloaded-line in dark mode
### Environment
**Vuetify Version:** 2.3.10
**Vue Version:** 2.6.12
**Browsers:** Chrome 84.0.4147.135
**OS:** Windows 10

### Steps to reproduce
Add a `VProgressCircular` to your project, change theme to dark.

### Expected Behavior
In light theme, both loaded part and unloaded part are visible - have different color from each other and from background color. 
Expected the same in dark mode.

### Actual Behavior
In dark mode, unloaded part isn't visible (it's almost black too)

### Reproduction Link
<a href="https://codepen.io/Tzahile/pen/qBZjQaK" target="_blank">https://codepen.io/Tzahile/pen/qBZjQaK</a>

### PR DESCRIPTION
**Pull request by bigbadfatman**
## Description
Tweaked the background color of circular progress bar

## Motivation and Context
Background color of circular progress bar clashes with the default app background color on the dark theme. Tweaked the background color to make it more visible.
fixes #12149 

## How Has This Been Tested?
Manual visual inspection

## Markup:
<details>

```vue
<template>
  <v-container>
    <VProgressCircular
      :value="10"
    />
  </v-container>
</template>

<script>
  export default {
    data: () => ({
    //
    }),
  }
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by bigbadfatman** [comment]
Hey guys, let me know if anything is missing from the PR.

File: packages/vuetify/src/components/VProgressCircular/VProgressCircular.sass [hunk 1]
```diff
@@ -36,7 +36,8 @@
     justify-content: center
 
   &__underlay
-    stroke: $progress-circular-underlay-stroke
+    stroke: currentColor
```
Comment (johnleider) [inline packages/vuetify/src/components/VProgressCircular/VProgressCircular.sass:39]: This is a breaking change.

File: packages/vuetify/src/components/VProgressCircular/_variables.scss [hunk 2]
```diff
@@ -3,5 +3,4 @@
 $progress-circular-rotate-animation: progress-circular-rotate 1.4s linear infinite !default;
 $progress-circular-rotate-dash: progress-circular-dash 1.4s ease-in-out infinite !default;
 $process-circular-intermediate-svg-transition: all .2s ease-in-out !default;
-$progress-circular-underlay-stroke: rgba(map-get($shades, 'black'), 0.1) !default;
```
Comment (johnleider) [inline packages/vuetify/src/components/VProgressCircular/_variables.scss:6]: This is a breaking change.

**Review by johnleider — CHANGES_REQUESTED** [review]
In regards to the fix in general, this is what my Playground.vue file looks like with the provided reproduction:

image

File: packages/vuetify/src/components/VProgressCircular/_variables.scss (hunk 2 above)
Comment (bigbadfatman) [reply packages/vuetify/src/components/VProgressCircular/_variables.scss:6]: Went back and realized it was only working on Firefox. Decided to switch out black for grey instead. Attached screenshots of how it looks now.

Screenshot 2020-10-05 at 18 09 26
Screenshot 2020-10-05 at 18 09 34

File: packages/vuetify/src/components/VProgressCircular/VProgressCircular.sass (hunk 1 above)
Comment (bigbadfatman) [reply packages/vuetify/src/components/VProgressCircular/VProgressCircular.sass]: Fixed in another commit

**Comment by bigbadfatman** [comment]
@johnleider Let me know if these changes work.

File: packages/vuetify/src/components/VProgressCircular/_variables.scss (hunk 2 above)
Comment (bigbadfatman) [reply packages/vuetify/src/components/VProgressCircular/_variables.scss:6]: @johnleider Screenshots above of the progressbar with grey as underlay color

**Review by KaelWD — APPROVED** [review]
It's a slightly darker grey than before when on a white background but is much more visible on a dark one.

**Comment by KaelWD** [comment]
Just FYI your git name/email don't match github, run `git config set user.email me@example.com` with the address your github account uses.

**Comment by bigbadfatman** [comment]
> It's a slightly darker grey than before when on a white background but is much more visible on a dark one.



> Just FYI your git name/email don't match github, run `git config set user.email me@example.com` with the address your github account uses.

Sure thing @KaelWD will do

**Pull request merged**